src/python/pants/backend/python/target_types.py

```diff
@@ -568,6 +568,25 @@ class PexIncludeToolsField(BoolField):
     )
 
 
+class RunInSandboxField(BoolField):
+    alias = "run_in_sandbox"
+    default = True
```

Comment:
Should this default to `False`? It increases the likelihood Pants Just Works, vs making you discover a niche field.